internal/cli/upgrade/upgrade.go

```diff
@@ -205,6 +205,7 @@ func (u *Upgrade) resolveInputParameters(opts *Options) error {
 	if opts.Parameters.IncreaseResourceLimits {
```

Comment:
**nit**
maybe we should extract it to dedicated method and use in both places?

sth what we already have for `opts.Parameters.ResolveVersion()`

Reply:
Yes, it will be in one of the follow-up tasks